### PR TITLE
Turns the maid costume's apron into a detachable accessory.

### PR DESCRIPTION
**Pull request by Expletive**
:cl:
add: Maid Costume aprons can now be detached and reattached to any uniform.
/:cl:

I thought this would be fun. With the apron attached, the Maid costume is identical in appearance, but if you take the apron off, then it's rather plain.

apronless maid costume:
maidapronless

### CODE REVIEW

**Comment by Cheridan** [comment]
slippery slope

**Comment by dannno** [comment]
neat idea

**Comment by Iamgoofball** [comment]
FUCKING

# SLIPPERY

# S L O P E

**Comment by dannno** [comment]
^imagine being this retarded

**Comment by Chronitonity** [comment]
Id rather not think about that

**Comment by Kraseo** [comment]
someone touched an article of clothing and separated it in 2 pieces, slippery slope!!!!!


goofball when's your next pr to remove X thing you don't like for the nth time this year

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -503,6 +503,12 @@
 	fitted = FEMALE_UNIFORM_TOP
 	can_adjust = 0
 
+/obj/item/clothing/under/maid/Initialize()
+	. = ..()
+	var/obj/item/clothing/accessory/maidapron/A = new /obj/item/clothing/accessory/maidapron(src)
```
Comment (Cyberboss) [inline code/modules/clothing/under/miscellaneous.dm]: `var/obj/item/clothing/accessory/maidapron/A = new(src)`

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Tokiko1** [comment]
You should fix the inconsistant sleeve lenght of the original sprite while at it.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**